docs/modules/servers/partials/operate/webadmin.adoc

```diff
@@ -2567,6 +2567,67 @@ Response codes:
 
 * 204: The quota has been updated to unlimited value.
 
+===== Domain DropList
```

Comment:
Missing the upper section declaration `Add entry to the DropList`?

Also missing global adding?

Reply:
Having a pass again for some reason it seems the droplist section aint present in the antora doc at all! I don't know if we missed that when introducing the droplist or if it got lost when we were refactoring the antora doc (if it's the case, sorry)

Then we probably need to add more globally the droplist section as well, as `Getting the DropList` and `Testing a denied entity existence` subsections, to have something similar to https://james.apache.org/server/manage-webadmin.html#administrating-droplists

You think you can help with that @Maxxx873 ?